wallet-new/test/unit/Wallet/Inductive/Validation.hs

```diff
@@ -201,15 +193,21 @@ inductiveIsValid Inductive{..} = do
 -------------------------------------------------------------------------------}
 
 instance (Hash h a, Buildable a) => Buildable (InductiveValidationError h a) where
-  build InductiveInvalidBoot{..} = bprint
+  build (InductiveInvalidBoot
+             inductiveInvalidBoot
+             inductiveInvalidError) = bprint
```

Comment:
👍 for removing uses of RecordWildcards. Maybe at some point we can go through and drop it from `default-extensions` / drop all uses.